website/content/v1.2/advanced/talos-api-access-from-k8s.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Talos API access from Kubernetes"
+description: "How to access Talos API from within Kubernetes."
+aliases:
+  - ../guides/talos-api-access-from-k8s
+---
+
+In this guide, we will enable the Talos feature to access the Talos API from within Kubernetes.
+
+## Enabling the Feature
+
+Edit the machine configuration to enable the feature, specifying the Kubernetes namespaces from which Talos API
+can be accessed and the allowed Talos API roles.
+
+```bash
+talosctl -n 172.20.0.2 edit machineconfig
+```
+
+Configure the `kubernetesTalosAPIAccess` like the following:
+
+```yaml
+spec:
+  machine:
+    features:
+      kubernetesTalosAPIAccess:
+        enabled: true
+        allowedRoles:
+          - os:reader
+        allowedKubernetesNamespaces:
+          - default
+```
+
+## Injecting Talos ServiceAccount into manifests
+
+Create the following manifest file `deployment.yaml`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: talos-api-access
+spec:
+  selector:
+    matchLabels:
+      app: talos-api-access
+  template:
+    metadata:
+      labels:
+        app: talos-api-access
+    spec:
+      containers:
+        - name: talos-api-access
+          image: alpine:3
+          command:
+            - sh
+            - -c
+            - |
+              wget -O /usr/local/bin/talosctl https://github.com/siderolabs/talos/releases/download/<talos version>/talosctl-linux-amd64
```

Comment:
we could use the hugo release variable here so that the talosctl version in the example doesn't need to be updated

Reply:
Doesn't `<talos version>` get updated automatically? I found it from the other places in docs.

Reply:
cool, i thought it needed a syntax like `{{` maybe I'm wrong :sweat_smile: